.grype.yaml

```diff
@@ -60,3 +68,11 @@ ignore:
     package:
       type: npm
       name: qs
+  - vulnerability: GHSA-p8p7-x288-28g6
```

Comment:
Looks like these two at the end might not be required any more?

POLICY_CONFIGURATION.md also needs updating.

Or it'd be nice to merge something similar to https://github.com/DEFRA/defra-docker-java/pull/21

Reply:
Unfortunately they still affect the Node 14 image so have kept them there. This will be the last Node 14 build, and I'll look replace that with Node 20!

Def would like to follow what you have on the Java side. I'll do that in the next release